visualization/app/codeCharta/core/scenario/scenarios.json

```diff
@@ -0,0 +1,83 @@
+[
+  {
+    "name": "Complexity",
+    "settings": {
+      "neutralColorRange": {
+        "from": 20,
+        "to": 40,
+        "flipped": false
+      },
+      "areaMetric": "rloc",
+      "heightMetric": "mcc",
+      "colorMetric": "mcc",
+      "deltas": false,
+      "amountOfTopLabels": 1,
+      "scaling": {
+        "x": 1,
+        "y": 1,
+        "z": 1
+      },
+      "camera": {
+        "x": 0,
+        "y": 300,
+        "z": 1000
+      },
+      "margin": 1,
+      "deltaColorFlipped": false
+    }
+  },
+  {
+    "name": "Coverage",
```

Comment:
We should have a distinctive name between the two coverage scenarios

Reply:
Oh, good point! I'll change that.